src/Reader/Extension/PodcastIndex/AttributesReader.php

```diff
@@ -0,0 +1,460 @@
+ *       role?: string,
+ *       group?: string,
+ *       img?: string,
+ *       href?: string
+ *     }
+ * @psalm-type TrailerObject = object{
+ *       title: string,
+ *       pubdate: string,
+ *       url: string,
+ *       length?: int,
+ *       type?: string,
+ *       season?: int
+ *     }
+ * @psalm-type RemoteItemObject = object{
+ *       feedGuid: string,
+ *       feedUrl?: string,
+ *       itemGuid?: string,
+ *       medium?: string,
+ *       title?: string
+ *     }
+ * @psalm-type ValueRecipientObject = object{
+ *       type: string,
+ *       address: string,
+ *       split: int,
+ *       name?: string,
+ *       customKey?: string,
+ *       customValue?: string,
+ *       fee?: bool,
+ *     }
+ * @psalm-type ValueTimeSplitObject = object{
+ *        startTime: int,
+ *        duration: int,
+ *        remoteStartTime?: int,
+ *        remotePercentage?: int,
+ *        valueRecipients?: list<ValueRecipientObject>,
+ *        remoteItem?: RemoteItemObject
+ *      }
+ * @psalm-type ValueObject = object{
+ *       type: string,
+ *       method: string,
+ *       suggested?: float,
+ *       valueRecipients: list<ValueRecipientObject>,
+ *       valueTimeSplits?: list<ValueTimeSplitObject>,
+ *     }
+ * @psalm-type ImagesObject = object{
+ *        srcset: string,
+ *      }
+ * @psalm-type DetailedImageObject = object{
+ *        href: string,
+ *        alt?: string,
+ *        purpose?: string,
+ *        type?: string,
+ *        aspectRatio?: string,
+ *        width?: int,
+ *        height?: int,
+ *      }
+ * @psalm-type SocialInteractObject = object{
+ *       protocol: string,
+ *       uri: string,
+ *       priority?: int,
+ *       accountId?: string,
+ *       accountUrl?: string,
+ *     }
+ * @psalm-type TranscriptObject = object{
+ *       url: string,
+ *       type: string,
+ *       language?: string,
+ *       rel?: string
+ *     }
+ * @psalm-type ChaptersObject = object{
+ *       url: string,
+ *       type: string
+ *     }
+ * @psalm-type SoundbiteObject = object{
+ *       title?: string,
+ *       startTime: string,
+ *       duration: string
+ *     }
+ * @psalm-type SeasonObject = object{
+ *         value: int,
+ *         name?: string
+ *       }
+ * @psalm-type EpisodeObject = object{
+ *         value: int|float,
+ *         display?: string
+ *       }
+ * @psalm-type SourceObject = object{
+ *        uri: string,
+ *        contentType?: string
+ *      }
+ * @psalm-type IntegrityObject = object{
+ *        type: string,
+ *        value: string
+ *      }
+ * @psalm-type AlternateEnclosureObject = object{
+ *        type: string,
+ *        length?: int,
+ *        bitrate?: int|float,
+ *        height?: int,
+ *        lang?: string,
+ *        title?: string,
+ *        rel?: string,
+ *        codecs?: string,
+ *        default?: bool,
+ *       sources: list<SourceObject>,
+ *       integrity?: IntegrityObject,
+ *      }
+ */
+class AttributesReader
+{
```

Comment:
As a static helper tool/utility, this class should not be exposed to users, we want to be able to break BC on classes like this whenever we feel like it.

This likely applies to the introduced classes `ElementGenerator` and `Validator` too.

```suggestion
 * }
 *
 * This class is internal to the library and should not be referenced by consumer code.
 * Backwards Incompatible changes can occur in Minor and Patch Releases.
 * 
 * @psalm-internal Laminas\Feed
 * @psalm-internal LaminasTest\Feed
 */
final class AttributesReader
{
```